cmd/mattermost-mattermod/main.go

```diff
@@ -25,16 +25,23 @@ func init() {
 }
 
 func main() {
+	var exitCode = 1
+	defer func() {
+		os.Exit(exitCode)
+	}()
```

Comment:
I am wondering if this is ever needed by something else or just something we did it like that? If not, I think it will be better to clean this `os.Exit` business and just simply return from `main`.